packages/terra-functional-testing/lib/cli/test-runner-cli.js

```diff
@@ -0,0 +1,13 @@
+const { program } = require('commander');
+const { TestRunner } = require('../test-runner');
+
+const run = async () => {
+  program
+    .option('--config <path>', 'A file path to the test runner configuration.');
```

Comment:
```suggestion
    .option('-c, --config <path>', 'A file path to the test runner configuration.');
```

Reply:
Added the shorthand cli option:

https://github.com/cerner/terra-toolkit/pull/406/commits/323a62a599edd71f5c21abcd5ba004bf9b8370eb